lifecycle-operator/chart/templates/keptnapp-crd.yaml

```diff
@@ -4,7 +4,10 @@ metadata:
   name: keptnapps.lifecycle.keptn.sh
   annotations:
     controller-gen.kubebuilder.io/version: v0.13.0
-    cert-manager.io/inject-ca-from: '{{ .Release.Namespace }}/keptn-certs'
```

Comment:
As the `annotations` in this place are hard-coded, we can simplify this - there is no need to use the variable `caAnnotations` and we can directly add `.Values.global.caInjectionAnnotations` here